tests/testthat/test-assignment_linter.R

```diff
@@ -186,9 +245,190 @@ test_that("multiple lints throw correct messages", {
     list(
       list(message = "Replace <<- by assigning to a specific environment", line_number = 2L),
       list(message = "Replace ->> by assigning to a specific environment", line_number = 3L),
-      list(message = "Use <-, not ->", line_number = 4L),
+      list(message = "Use <- for assignment, not ->", line_number = 4L),
+      list(message = "Avoid the assignment pipe %<>%", line_number = 5L)
+    ),
+    assignment_linter(operator = "<-")
+  )
+})
+
+test_that("assignment operator can be toggled", {
+  eq_linter <- assignment_linter(operator = "=")
+  any_linter <- assignment_linter(operator = "any")
```

Comment:
Maybe add a test that lints with this config, too?

Reply:
Well, this specific config won't throw any lints as of now.

I added a test of `operator="any", allow_trailing=FALSE`, though.